### ISSUE TITLE
Provide a list of where all the pagers are in the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15804]
## Description
I need a list of everywhere a pager is in the CMS so I can then either change it to the proper heading level or provide the correct heading level for an engineer to change.

## Acceptance Criteria
- [x] Provide a list to Laura of everywhere the Pager shows in the CMS
- [x] If this can be done ONLY in the CMS, meaning no other engineering changes need to occur, provide instructions for how to change the heading level of the pager

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] Accelerated Publishing

**Comment by BerniXiongA6** [comment]
Note to @BerniXiongA6 -- see Slack thread that [originated this ticket](https://dsva.slack.com/archives/CT4GZBM8F/p1698093314073169). Prioritize this ticket for refinement.

**Comment by tonytaylor** [comment]
A report is available for perusal at: https://docs.google.com/spreadsheets/d/1StwksIw6BG13kNwnob9y39mV4kBeM8I6Fds5f4nQ9Lw/edit?usp=sharing.

 
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/69f3b106-c164-46d5-ac70-3d99cf7a70de)

## About Report Entries

There are some entries that don't rely on a pager that may be configurable (N/Y).  These can be ignored along with entries
that don't use pagers and are not configurable (N/N).

All entries that uses a pager and is not configurable (Y/N) will require an engineer to update the underlying template.

For all entries that uses a pager and is configurable (Y/Y), updating its heading is as follows:

- Go to Edit/Config URL
- Click on the Pager Options Configuration link ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/d512add5-2ba8-4875-b6ee-2f4140d52f76)
- The _Heading Level_ configuration option should be present ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/69f3b106-c164-46d5-ac70-3d99cf7a70de)
- Choose a heading level, click `Apply`, and then click `Save`
- Either scroll down to verify via Preview pager area, or navigate to Display URL and locate the pager near the bottom of the page
- Right-click on or near the first link in the pager and select Inspect. The invisible header that contains the pager--its element should have been updated to reflect your selection. ![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/33521f07-2756-4e84-8182-0b8b04b6d44d)

## About PostAPI Queue

The PostAPI Queue page has a pager, but the queue isn't initially populated.

- Go to /admin/config/va-gov-post-api/facility-force-queue 
![20231107--1420](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/5f9e3ec6-4e1d-4476-97a4-cc62b6943aa0)
- Select an entry and click `Force-Queue Selected Items`
- Go to /admin/config/post-api/queue ![20231107--1422](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/3f0935ac-4684-4056-b649-d6114a1729d2)
- The page should be populated similar to above and the pager can be inspected.

**Comment by laflannery** [comment]
@tonytaylor I tried to go to the Config URL for the main content view and it doesn't exist - can you take a look:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/9ec3d952-9e3e-4829-984e-a1bd11bc970f)

**Comment by tonytaylor** [comment]
Looking into it.

**Comment by tonytaylor** [comment]
I think views may have its configuration disabled in production.  Gah!

Apologies for not catching this sooner.  This may need to be done in configuration.  cc: @ndouglas

**Comment by ndouglas** [comment]
Yeah, `views_ui` is disabled in production. Any changes need to be made in the form of a PR.

If I'm understanding correctly, perhaps a course of action would be:
- Laura going e.g. here: https://main-8ry6zt2asg946vdfuiryyamuc9gkuyzc.demo.cms.va.gov/admin/structure/views/view/content and noting what changes need to be made, and then 
- Tony opening a PR for the changes?

**Comment by laflannery** [comment]
@ndouglas There are A LOT of places tht have a pager. @tonytaylor put a [list together here](https://docs.google.com/spreadsheets/d/1StwksIw6BG13kNwnob9y39mV4kBeM8I6Fds5f4nQ9Lw/edit#gid=0).

What I think I should do it update this sheet with a column of what the header level should be and then Tony can go make a PR (or multiple depending) to update everything that needs to be updated. Does that make sense to both of you?

**Comment by ndouglas** [comment]
Would you like me to open a followup issue to cover the work of updating the spreadsheet?

**Comment by laflannery** [comment]
I might have just added a new AC... But ya maybe that's the better way to do it

**Comment by ndouglas** [comment]
Whatever y'all wanna do 🤷🏻

**Comment by BerniXiongA6** [comment]
Yes please can we add a new ticket @ndouglas @laflannery -- cleaner that way.
